WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/CardReaderSettingsConnectedViewController.swift

```diff
@@ -150,18 +169,23 @@ private struct Section {
 }
 
 private enum Row: CaseIterable {
-    case temporaryHeader
+    case connectedReader
+    case disconnectButton
 
     var type: UITableViewCell.Type {
         switch self {
-        case .temporaryHeader:
-            return TitleTableViewCell.self
+        case .connectedReader:
+            return ConnectedReaderTableViewCell.self
+        case .disconnectButton:
+            return ButtonTableViewCell.self
         }
     }
 
     var height: CGFloat {
         switch self {
-        case .temporaryHeader:
+        case .connectedReader:
+            return 60
```

Comment:
Could this be also .automaticDimension? (It's a genuine question, not a veiled suggestion 😉 )

Reply:
I had it that way and the height wasn't enough. I'll try tinkering with the cell constraints in the reusable view